esp/services/ws_store/ws_storeService.cpp

```diff
@@ -358,7 +364,11 @@ bool CwsstoreEx::onFetchKeyMetadata(IEspContext &context, IEspFetchKeyMDRequest
         {
             Owned<IEspKVPair> kvpair = createKVPair("","");
 
-            kvpair->setKey(attributes->queryName());
+            //it's possible this has been encoded, so decode it
+            StringBuffer decoded;
+            decodePtreeName(decoded, attributes->queryName());
```

Comment:
The attribute names have not been encoded - so there should not be matching code to decode them.

Reply:
The IPT attributes iterator contains all keys under a particular namespace branch, those keys would have been encoded in the set method. I believe decoding is necessary here.

Reply:
When I read the code it looked like the attributes would be things like DALI_KVSTORE_CREATEDTIME_ATT which have not been encoded (and are also not user specified).

Reply:
@rpastrana you haven't responded to this comment.  I can merge as-is because the decoding will have no effect, but I also don't think it is ever necessary.

Reply:
no need to decode. Will revert